src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -24,11 +24,65 @@
  */
 package com.sonyericsson.hudson.plugins.gerrit.trigger.hudsontrigger;
 
-import com.google.common.collect.Iterators;
```

Comment:
Please put back the original order of imports.
(Or is this the original but I modified it in my (base) commit?)

Reply:
I won't review this file for now, at least until the formatting diffs are gone :)

Reply:
Missed that :) but I reverted the formatting now.